app-backend/batch/src/main/scala/stacExport/StacExtent.scala

```diff
@@ -1,17 +1,27 @@
 package com.rasterfoundry.batch.stacExport
 
-import io.circe.Encoder
+import io.circe._
+import io.circe.generic.semiauto._
 
 // Spatial = bbox[lowerleftx, lowerlefty, upperrightx, upperrighty]
 // temportal = ["start date isostring"||null, "end date isostring"||null]
 // https://github.com/radiantearth/stac-spec/blob/dev/collection-spec/collection-spec.md#extent-object
-case class StacExtent(spatial: List[Double], temporal: List[Option[String]])
+case class StacExtent(spatial: SpatialExtent, temporal: TemporalExtent)
+
+case class SpatialExtent(bbox: List[List[Double]])
```

Comment:
You can use types from gt-server to be more precise here -- https://github.com/geotrellis/geotrellis-server/blob/c20165867eadb376e69bc3b2f6170b88347d5230/stac/src/main/scala/package.scala#L14-L19 or https://github.com/geotrellis/geotrellis-server/blob/c20165867eadb376e69bc3b2f6170b88347d5230/stac/src/main/scala/package.scala#L28

Reply:
I think this is a very good point. But circe seems to have a hard time decoding it. Per offline discussions, I will keep the current implementation. Let's enhance it in future issues.